### PR TITLE
[Python] Mentor - Add Lorne Hyde

### PR DESCRIPTION
**Pull request by LorneHyde**

### CODE REVIEW

File: mentors/python/l.json [hunk 1]
```diff
@@ -31,4 +31,12 @@
...
     "bio": "I love python and is the main programming language that I use in everyday, it's easy and beautiful language. I want to help everyone write more pythonic codes!"
   }
+  {
+  "github_username": "LorneHyde",
+  "name": "Lorne Hyde",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "My passion for coding began with python, and it led me to choosing to study Computer Science at university! I hope to share my love of python with others."
+}
```
Comment (rpottsoh) [inline mentors/python/l.json]: The lines I have hi-lighted need to each be indented 2 more spaces.  The formatting of your bio information should appear like the entries that precede yours.

File: mentors/python/l.json [hunk 2]
```diff
@@ -31,4 +31,12 @@
     "avatar_url": null,
     "bio": "I love python and is the main programming language that I use in everyday, it's easy and beautiful language. I want to help everyone write more pythonic codes!"
   }
```
Comment (rpottsoh) [inline mentors/python/l.json]:
```suggestion
  },
```

**Comment by rpottsoh** [comment]
@LorneHyde this is just s friendly reminder that you have an open PR.  If you have questions we're happy to help.

**Comment by BethanyG** [comment]
I am going to close this due to 12+ months of inactivity.  @LorneHyde, if you would still like to submit this PR, please ping one of us, and we'd be happy to re-open or re-constitute it.

**Pull request closed without merging**